lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1160,11 +1160,11 @@ def _create_node_runbook(
             azure_node_runbook.name = truncate_keep_prefix(
                 f"{name_prefix}-n{index}", 50
             )
-            # It's used as computer name only. Windows doesn't support name more
-            # than 15 chars
-            azure_node_runbook.short_name = truncate_keep_prefix(
-                azure_node_runbook.name, 15
-            )
+        # It's used as computer name only. Windows doesn't support name more
+        # than 15 chars
+        azure_node_runbook.short_name = truncate_keep_prefix(
+            azure_node_runbook.name, 15
```

Comment:
There is the place to keep first 5 chars. Other parts should keep as is.

Reply:
and I believe it also needs to be changed at https://github.com/microsoft/lisa/blob/main/lisa/sut_orchestrator/azure/platform_.py#L1160 as `name_prefix` is not always guaranteed to start with "lisa-"